### PR TITLE
Initial ahoy/script re-work for readability/re-use

### PR DESCRIPTION
**Pull request by tobybellwood**
I've gone through the scripts and ahoy file and tried to make it a little more streamlined, added some additional context comments.  I've also reused the ahoy commands with switched parameters per branch - I think it makes sense?

### CODE REVIEW

File: .ahoy.yml [hunk 1]
```diff
@@ -23,79 +23,78 @@ commands:
...
+        ahoy satis-build satis-config/govcms-stable.json app
+      else
+        echo -e "\033[1;35m--> Building govcms-"$1".json  \033[0m"
+        ahoy satis-build satis-config/govcms-"$1".json app/"$1"
+      fi
+
+  build-all:
+    usage: Build all satis package lists
     cmd: |
       ahoy build-stable
```
Comment (simesy) [inline .ahoy.yml:39]: Does `ahoy build-stable` need to be `ahoy build stable`?

File: scripts/check-dupes.sh [hunk 2]
```diff
@@ -30,6 +31,8 @@ do
             cat $file | jq -r '.packages | .["'"${project}"'"] | to_entries[] | "    \"\(.value.name)\": \"\(.key)\","'
         done
         echo '}'
+    else
+        echo -e "\033[1;35m--> No duplicates found!  \033[0m"
```
Comment (simesy) [inline scripts/check-dupes.sh:35]: Ooh unlikely but fair enough :P

**Comment by simesy** [comment]
Looks crisp. Since there is a new release of scaffold-tooling and require-dev, i'll test a re-roll with this branch.

**Comment by simesy** [comment]
I've tested this on https://github.com/govCMS/satis/pull/16 and fixed the build-all commands.

File: .ahoy.yml (hunk 1 above)
Comment (simesy) [reply .ahoy.yml:39]: Oh i guess its likely once you get the config right..

**Comment by tobybellwood** [comment]
To be merged as part of #16

**Pull request closed without merging**